.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
```

Comment:
There is debate as to whether you should commit your .vscode file to version control.  Generally it is things related to your own personal setup, and you don't want to force other users to share them.  Google .gitignore to discover how to exclude files or folders from version control.

Sometimes you do want local configuration to be shared, but only do it intentionally!